src/main/java/seedu/address/logic/Messages.java

```diff
@@ -13,11 +13,22 @@
 public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX = "The company index provided is invalid";
+    public static final String MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX =
+        "Oops! It seems you have entered an index that is greater than the number of companies that you have. "
```

Comment:
suggestion? 

Oops! You have entered an index greater than the number of companies shown in the list.